src/types/NewTodo.ts

```diff
@@ -0,0 +1,5 @@
+export interface NewTodo {
+  title: string,
+  userId: number,
```

Comment:
Consider removing the userId property, as it is not required for the task description.

Reply:
done